oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerAuthenticationManagerResolver.java

```diff
@@ -130,6 +165,10 @@ public AuthenticationManager resolve(HttpServletRequest request) {
 		return authenticationManager;
 	}
 
+	public void setIssuerConverter(Converter<HttpServletRequest, String> issuerConverter) {
```

Comment:
Please add JavaDoc, including a `@since 5.5` so it's clear when the method became available.